en/docs/use-cases/streaming-tutorials/working-with-kafka.md

```diff
@@ -100,40 +98,38 @@ Let's create a basic Siddhi application to consume messages from a Kafka topic.
     ```
 
 !!! info
-    You just created a Siddhi application that listens to a Kafka topic named `productions` and logs any incoming messages. When logging, the name attribute of the message is converted to upper case. However, you have still not created this Kafka topic or published any messages to it. To do this, proceed to the next section.
-
-
-#### Generate Kafka messages
+    You just created a Siddhi application that listens to a Kafka topic named `productions` and logs any incoming messages. When logging, the name attribute of the message is converted to upper case. However, you have still not created this Kafka topic or published any messages to it. To do this, proceed to the next substep.
 
-Now let's generate some Kafka messages that the Streaming Integrator can receive. 
 
-1. First, let's create a topic named `productions` in the Kafka server. To do this, navigate to `<KAFKA_HOME>` and run following command:
+3. Generate some Kafka messages that the Streaming Integrator can receive by following the procedure below:
 
-    ```
-    bin/kafka-topics.sh --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic productions
-    ```
-
-2. Now let's run the Kafka command line client to push a few messages to the Kafka server.
-
-    ```
-    bin/kafka-console-producer.sh --broker-list localhost:9092 --topic productions
-    ```
-
-3. Now you are prompted to type messages in the console. Type the following in the command prompt:
-
-    ```
-    {"event":{ "name":"Almond cookie", "amount":100.0}}
-    ```
-
-   This pushes a message to the Kafka Server. Then, the Siddhi application you deployed in the Streaming Integrator consumes this message. As a result, the Streaming Integrator log displays the following:
-
-    ```
-    INFO {io.siddhi.core.stream.output.sink.LogSink} - HelloKafka : OutputStream : Event{timestamp=1562069868006, data=[ALMOND COOKIE, 100.0], isExpired=false}
-    ```
-
-You may notice that the output message has an uppercase name: `ALMOND COOKIE`. This is because of the simple message transformation done in the Siddhi application.   
+    1. First, let's create a topic named `productions` in the Kafka server. To do this, navigate to `<KAFKA_HOME>` and run following command:
```

Comment:
Avoid saying "First, let's", just make it "Create ...." because for one you are using numbering to show the sequence and secondly you are providing the user a certain instruction that they need to follow.

Please apply this comment for the following sections as well were you start each step by saying "Now".

Reply:
Changed as specified